.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -24,11 +24,63 @@ labels: bug
 4. See error
 
 ### Your personal set up
-<!-- Tell us a little about the system you're using. -->
+<!--
+Tell us a little about the system you're using.
+Please include information about how you installed,
+e.g. are you using a distribution such as zero-to-jupyterhub or the-littlest-jupyterhub.
+ -->
 
  - OS:
- <!-- [e.g. linux, OSX] -->
- - Version:
- <!-- e.g. jupyterhub --version. --->
- - Configuration:
- <!-- Be careful not to share any sensitive information. --->
+ <!-- [e.g. ubuntu 20.04, macOS 11.0] -->
+ - Version(s):
+ <!-- e.g. jupyterhub --version, python --version --->
+
+<!-- For reproduction, it's useful to have the full environment. For example, the output of `pip freeze` or `conda list` --->
+
+<details>
+```
+# paste output of `pip freeze` or `conda list` here
+```
+</details>
+
+- Configuration:
+<!--
+For JupyterHub, especially include information such as what Spawner and Authenticator are being used.
+Be careful not to share any sensitive information.
+
+You can paste jupyterhub_config.py below.
+To exclude lots of comments and empty lines from auto-generated jupyterhub_config.py, you can do:
+
+    grep -v '\(^#\|^[[:space:]]*$\)' jupyterhub_config.py
+-->
+
+<details>
+```python
+# jupyterhub_config.py
+```
+</details>
+
+- Logs:
+
+<!--
+Errors are often logged by jupytehub. How you get logs depends on your deployment.
+With kubernetes it might be:
+
+    kubectl get pod # hub pod name starts with hub...
+    kubectl logs hub-...
+    # or for a single-user server
+    kubectl logs jupyter-username
+
+Or the-littlest-jupyterhub:
+
+    journalctl -u jupyterhub
+    # or for a single-user server
+    journalctl -u jupyter-username
+
+-->
+
+<details>
+```
+# paste relevant logs here, if any
+```
+</details>
```

Comment:
``````suggestion
- <details><summary>Logs</summary>
<!--
Errors are often logged by jupytehub. How you get logs depends on your deployment.
With kubernetes it might be:

    kubectl get pod # hub pod name starts with hub...
    kubectl logs hub-...
    # or for a single-user server
    kubectl logs jupyter-username

Or the-littlest-jupyterhub:

    journalctl -u jupyterhub
    # or for a single-user server
    journalctl -u jupyter-username
-->

```
# paste relevant logs here, if any
```
</details>
``````